frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -104,11 +104,25 @@ const footerVariantStyles = {
   `,
 };
 
-function getParameterPanelBackgroundColor(theme?: DisplayTheme) {
+function getParameterPanelBackgroundColor(
+  theme: DisplayTheme | undefined,
+  isSticky: boolean,
+) {
   if (theme === "night") {
-    return color("bg-black");
+    return `color-mix(in srgb, var(--mb-color-bg-black), var(--mb-color-embed-bg-color-override, var(--mb-color-bg-black))  ${
+      isSticky ? 15 : 100
+    }%)`;
   }
-  return color("bg-white");
+
+  if (theme === "transparent") {
+    return `color-mix(in srgb, var(--mb-color-bg-white), transparent  ${
+      isSticky ? 15 : 100
+    }%)`;
+  }
```

Comment:
I added this block for completeness. Actually, this should be done in `Handle a case when `theme=transparent` and `background=false` are supplied` in [the epic](https://github.com/metabase/metabase/issues/43838), when I'm ensuring the `theme=transparent` behavior is still working.